docs/app/css/style.css

```diff
@@ -669,7 +626,6 @@ ul.buckets li a {
   font-weight: 500;
   padding: 16px 0;
   text-decoration: none;
-  -webkit-transition: all 0.45s cubic-bezier(0.35, 0, 0.25, 1);
```

Comment:
FYI: This was intentionally removed, because it will be set in the selector below (this was an unnecessary property)